src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphMethodConfigurationDAO.scala

```diff
@@ -81,52 +43,15 @@ class GraphMethodConfigurationDAO extends MethodConfigurationDAO with GraphDAO {
       .getOrElse(throw new IllegalArgumentException("Cannot save entity to nonexistent workspace " + workspaceNamespace + "::" + workspaceName))
 
     def methodConfigPipe = methodConfigPipeline(graph, workspaceNamespace, workspaceName, methodConfiguration.namespace, methodConfiguration.name)
-
-    // get the methodConfigVertex, creating if it doesn't already exist
-    val (configVertex, inputsVertex, outputsVertex, prerequisitesVertex) =
-      getSinglePipelineResult(methodConfigPipe) match {
-        case None => {
-          val configVertex = addVertex(graph, null)
-          workspace.addEdge(MethodConfigEdgeType, configVertex)
-
-          val (inputsVertex, outputsVertex, prerequisitesVertex) = createSubVertices(graph, configVertex)
-
-          (configVertex, inputsVertex, outputsVertex, prerequisitesVertex)
-        }
-
-        case Some(configVertex) => {
-          configVertex.getVertices(Direction.OUT).foreach(_.remove())
-          val (inputsVertex, outputsVertex, prerequisitesVertex) = createSubVertices(graph, configVertex)
-          (configVertex, inputsVertex, outputsVertex, prerequisitesVertex)
-        }
-      }
-
-    setVertexProperties(methodConfiguration, configVertex)
-
-
-    Option(methodConfiguration.methodStoreConfig).map(_ => {
-        configVertex.setProperty("methodConfigNamespace", methodConfiguration.methodStoreConfig.methodConfigNamespace)
-        configVertex.setProperty("methodConfigName", methodConfiguration.methodStoreConfig.methodConfigName)
-        configVertex.setProperty("methodConfigVersion", methodConfiguration.methodStoreConfig.methodConfigVersion)
-      })
-    configVertex.setProperty("methodNamespace", methodConfiguration.methodStoreMethod.methodNamespace)
-    configVertex.setProperty("methodName", methodConfiguration.methodStoreMethod.methodName)
-    configVertex.setProperty("methodVersion", methodConfiguration.methodStoreMethod.methodVersion)
-
-    methodConfiguration.inputs.foreach { entry => inputsVertex.setProperty(entry._1, entry._2) }
-    methodConfiguration.outputs.foreach { entry => outputsVertex.setProperty(entry._1, entry._2) }
-    methodConfiguration.prerequisites.foreach { entry => prerequisitesVertex.setProperty(entry._1, entry._2) }
+    val configVertex = getSinglePipelineResult(methodConfigPipe) match {
```

Comment:
some places we do getOrElse, some like this, I prefer getOrElse pattern for this